lib/search_solr_tools/translators/nsidc_json.rb

```diff
@@ -54,7 +55,8 @@ def translate(json_doc)
           'facet_sponsored_program'   => translate_short_long_names_to_facet_value(json_doc['internalDataCenters']),
           'facet_temporal_resolution' => translate_temporal_resolution_facet_values(json_doc['parameters']),
           'facet_spatial_resolution'  => translate_spatial_resolution_facet_values(json_doc['parameters']),
-          'sponsored_programs'        => translate_internal_datacenters(json_doc['internalDataCenters'])
+          'sponsored_programs'        => translate_internal_datacenters(json_doc['internalDataCenters']),
+          'facet_featured'            => translate_featured(json_doc, spatial_coverages)
```

Comment:
The "Spatial Coverage" facet still exists (line 42 above); however, we won't be exposing it via DSS anymore.  Do you think I should just get rid of the other facet?

Reply:
I'm a bit conflicted but yes, we probably should clean it up so it's not adding clutter. (I assume by "the other facet" you mean "Spatial Coverage.")

Reply:
Yes, sorry, I should have been more clear.

I'll go ahead and remove that facet then.  I just left it in (temporarily) because there are a few other facets that we've left in but aren't using, but at the same time, the functionality from the Spatial Coverage facet is being preserved in the new Featured facet anyway, so less need for duplication.